### ISSUE TITLE
Will you release the training code?

### ISSUE THREAD

**Issue by zengshao0622** [zhangy76/KNOWN-Hand#2]
Hi Yufei,

Thank you so much for sharing the nice work on the ECCV paper! The idea really interests me a lot. I wonder if you will release the training code?

Thank you!

**Comment by zengshao0622** [comment]
> Hi Yufei,
> 
> Thank you so much for sharing the nice work on the ECCV paper! The idea really interests me a lot. I wonder if you will release the training code?

**Comment by zhangy76** [comment]
Thanks for your interest. Currently, I am unable to release the training code. However, if you have an established training pipeline, you can directly incorporate the loss, MANO, and regression model (all released here) into your setup in a straightforward manner. Please feel free to reach out again if you encounter any issues.
